doc/manual/en/config/widgets/plugins/strftime/index.rst

```diff
@@ -0,0 +1,93 @@
+.. _strftime:
+
+The strftime plugin
+===================
+
+.. api-doc:: strftime
+
+Description
+-----------
+
+.. ###START-WIDGET-DESCRIPTION### Please do not change the following content. Changes will be overwritten
+
+This plugins integrates formated date and clock strings into based on strftime.
+
+Thanks to Michael Markstaller for implementing the jqclock plugin as reference.
```

Comment:
Actually the content between `.. ###START-WIDGET-DESCRIPTION###` and `.. ###END-WIDGET-DESCRIPTION###` gets replaced on every build with the content taken from the source. So removing that from source should be fine.